components/Main.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { Text, View } from 'react-native';
 import styles from '../styles';
+import Forum from './Forum'
 
 export default class App extends React.Component {
   render() {
     return (
       <View style={styles.container}>
+        <Forum/>
```

Comment:
Same goes for this change. On the master branch, the correct components should go into main, and Forum won't be the first screen.